### DISCUSSION TITLE
Styling in app notifications modals

### DISCUSSION THREAD

**Discussion by juliovillalvazo** [invertase/react-native-firebase#7496] [Q&A]
I'm trying to style the in app notifications modals and cards displayed with the in-app-messaging library, but I haven't found any documentation regarding this topic. Can someone help me with documentation on how I can modiify the button style and the modal in general? 🙏🏽

**Comment by davidepalazzo** [comment]
After the [initial setup](https://rnfirebase.io/in-app-messaging/usage#installation), you can customise your In-App Messaging elements (card, modal, etc.) within a campaign in the Firebase Console as mentioned in the [usage](https://rnfirebase.io/in-app-messaging/usage#usage) section. Navigate to Messaging from the sidebar, and then select New campaign > In-app messaging. At this point, you can choose a layout, content, and style of the message.
